pkgs/development/python-modules/pyemvue/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+
+# build-system
+, hatchling
+, build
+
+# propagated modules
+, requests
+, dateutil
+, python-jose
+, pycognito
+, typing-extensions
+}:
+
+buildPythonPackage rec {
+  pname = "pyemvue";
+  version = "0.18.0";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-s1Uuoiog3OkqOb6PSTqh79vgyLDGFIuZMCyQKH+qG3g=";
+  };
+
+  nativeBuildInputs = [
+    hatchling
+    build
+  ];
+
+  propagatedBuildInputs = [
+    requests
+    dateutil
+    python-jose
+    pycognito
+    typing-extensions
```

Comment:
```
Requires-Dist: pycognito>=2021.3.1
Requires-Dist: python-dateutil>=2.8.2
Requires-Dist: requests>=2.26.0
Requires-Dist: typing-extensions>=4.0.1
```

- remove `python-jose`
- rename `dateutil` to `python-dateutil`, also in the function argument

Reply:
Thanks! I am not sure why I added python-jose, but how did you produce the report that was included? It looks like it lists the actual imports versus what was specified.